tests/Bundle/Resources/App/Api/Entity/Book.php

```diff
@@ -5,12 +5,14 @@
 namespace AutoMapper\Tests\Bundle\Resources\App\Api\Entity;
 
 use ApiPlatform\Metadata\ApiResource;
+use AutoMapper\Attribute\Mapper;
 use AutoMapper\Attribute\MapTo;
 use AutoMapper\Tests\Bundle\Resources\App\Api\Processor\BookProcessor;
 use AutoMapper\Tests\Bundle\Resources\App\Api\Provider\BookProvider;
 
 /** A book. */
 #[ApiResource(provider: BookProvider::class, processor: BookProcessor::class)]
+#[Mapper(source: 'array', target: 'array')]
```

Comment:
So it means we can have a book **from** an array and **to** an array ?

Reply:
It means we can map to or from an array